chingsley/node-auth2-guided
# - implement session storage for cookies

#### What does this PR do?
It implements the functionality to enable session storage so that cookie is still valid after server restart provided the session has not expired.

#### Description of Task to be completed?
- Install npm package connect-session-knex.
- In the session config, setup the `store` property to enable session storage.
- test the implementation

#### How should this be manually tested?
- *In postman*, log in with a POST request to the endpoint `http://localhost:3000/api/auth/login`, providing your `username` and `password` in the request body. This action generates a cookie for your session.
- Confirm that after a successful login that you have access to the resource GET `http://localhost:3000/api/users`
- Without logging out, restart the server. Confirm that you still have access to GET /users after server restart (provided your session has not expired).

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
